website/source/api/acl/acl.html.md

```diff
@@ -285,6 +285,12 @@ The table below shows this endpoint's support for
 | ---------------- | ----------------- | ------------- | ------------ |
 | `NO`             | `none`            | `none`        | `none`       |
 
+-> **Note** - [ACL token
```

Comment:
Possible re-wording for the first sentence:

"To use the `login` process to create tokens in any connected secondary datacenter, ACL replication must be enabled. "

Reply:
I like it.